src/gui/widgets/text_box_base.cpp

```diff
@@ -478,13 +478,28 @@ void text_box_base::signal_handler_sdl_key_down(const event::ui_event event,
 
 // For copy/paste we use a different key on the MAC. Other ctrl modifiers won't
 // be modifed seems not to be required when I read the comment in
-// widgets/textbox.cpp:516. Would be nice if somebody on a MAC would test it.
-#ifdef __APPLE__
-	const unsigned copypaste_modifier = KMOD_LGUI | KMOD_RGUI;
-#else
-	const unsigned copypaste_modifier = KMOD_CTRL;
+// widgets/textbox.cpp:495. But I encourage changing the leader key to 'cmd'
+// for 'a' (select all) on macOS, to have all functionality in text boxes
+// aligned in a same direction, i.e. macOS defaults.
+
+// The election for leader key is not pervasive and, even though configurable
+// (or at least it _should_ be configurable), is very often set to what the
+// desktop environment defines as idiomatic.
+
+#ifndef __APPLE__
+	// Idiomatic leader key in GNU/Linux desktop environments and
+	// Microsoft desktop environments.
+	const unsigned leader_key = KMOD_CTRL;
```

Comment:
I don't think this should be called `leader_key`. Something like `modifier_key` would be better.